tests/Detector.Tests/DotNetCore/DotnetCoreDetectorTest.cs

```diff
@@ -49,6 +49,36 @@ public void Detect_ReturnsNull_IfProjectFile_DoesNotHaveTargetFrameworkSpecified
             Assert.Null(result);
         }
 
+        [Theory]
+        [InlineData("net6.0", true)]
+        [InlineData("net5.0", false)]
+        public void Detect_ReturnsExpected_BlazorWebAssemblyApp_ProjectFileHaveTargetFrameworkSpecified(
```

Comment:
i think it will be better if we write the tests like following ..

[InlineData("net6.0-preview5", true)]
[InlineData("net6.0-preview4", false)]
[InlineData("net5.0", false)]